### ISSUE TITLE
Download speed limiter

### ISSUE THREAD

**Issue by BaabakM** [telegramdesktop/tdesktop#24519]
### Is your feature request related to a problem?

In windows or android version of the app when a file is downloading it takes all bandwidth of the internet connection for that device.

### Describe the solution you'd like

a feature that limits the download speed of the application releases extra space to use the internet in other apps. Especially in Windows version, this feature is needed so much because when a file(s) is downloading in Telegram other programs actually stop working when it requires an internet connection!

### Describe alternatives you've considered

add a "pause" button for an occasional halt and later downloading.

### Additional context

make a download center for better productivity in downloading files and future features.

**Comment by mmortal03** [comment]
I just noticed this, as well. On Windows, when downloading a large file from Telegram, webpages will no longer load in Chrome, for instance.

**Comment by BaabakM** [comment]
> I just noticed this, as well. On Windows, when downloading a large file from Telegram, webpages will no longer load in Chrome, for instance.

exactly, I need some bandwidth to work with that alongside downloading files from telegram
